templates/docs.hbs

```diff
@@ -360,6 +360,6 @@
                 line-height: {{this.lineHeight}};
                 letter-spacing: {{this.letterSpacing}};
                 text-indent: {{this.paragraphSpacing}};">

```

Comment:
Also here - specifying all css properties for non existing preview doesn't make sense.